driver-core/src/main/com/mongodb/MongoCredential.java

```diff
@@ -706,9 +728,11 @@ public interface IdpInfo {
     }
 
     /**
-     * The response produced by an OIDC Identity Provider.
+     * The OIDC credential information.
+     *
+     * @since 5.1
      */
-    public static final class OidcCallbackResult {
+    public static final class OidcTokens {
```

Comment:
In the spec, an [example](https://github.com/mongodb/specifications/blob/master/source/auth/auth.md?plain=1#L1433) is given that uses the name `OIDCCredential`, but this would be confusing here, since we use the same terminology for creating an OIDC MongoCredential, `#createOidcCredential`.

Reply:
Rename back to OidcCallbackResult